### DISCUSSION TITLE
Add the NewFormClientSideComponentId and related properties to the ContentType model

### DISCUSSION THREAD

**Discussion by tomasz-sintel** [pnp/pnpcore#1420] [Ideas]
When developing an SPFx form customizer, the component needs to be connected to the list via properties in the content type ([reference](https://pnp.github.io/script-samples/spo-add-formextension-to-list/README.html?tabs=pnpps)).
Those properties (e.g. NewFormClientSideComponentId) are not available in the ContentType class.
Having to have some plug "on the side" in the form of a REST or PS script is not elegant.
My request is to add those properties to the ContentType class.

**Comment by jansenbe** [comment]
That makes sense @tomasz-sintel , I've created an issue to track this ask
